src/remoteSession.ts

```diff
@@ -67,7 +67,7 @@ export class RemoteSession extends Disposable {
 		try {
 			const useLocalSSH = await this.experiments.getUseLocalSSHProxy();
```

Comment:
If it failed to test local ssh and with feature flag enabled, then other code may not run